server/route.go

```diff
@@ -25,12 +25,12 @@ func drawRoutes(r *gin.Engine, appContext *system.AppContext) {
 		r.GET("/", route.Ping)
 		r.GET("/ping", route.Ping)
 		r.GET("/user", route.GetCurrentUser)
-		r.GET("/oauth/github/authorization", route.Authorize)
-		r.POST("/oauth/github/authentication", route.Authenticate)
+		r.GET("/oauth/github/authorization", route.GetAuthorization)
+		r.POST("/oauth/github/authentication", route.GetAuthentication)
```

Comment:
Nope, `r.POST("...", route.Authenticate)` remain unchanged. This action is handled by `POST` request. I only suggest the `route.Authorize` handler should be appropriate name as a verb indicating `GET` request handler.